Makefile

```diff
@@ -85,16 +85,27 @@ e2e: node_modules
 		cp config/config.json config/config-backup.json && cp config/default.json config/config.json || \
 		echo "config.json not found" && cp config/default.json config/config.json
 
+	@echo E2E: Starting the server
+	cd $(BUILD_SERVER_DIR) && $(MAKE) run
```

Comment:
This will build the client too. Not sure if that's intended? Also it's going to start the server asynchronously, does cypress wait for it to start?

Reply:
Yes, that's intended.  
Cypress will not wait but it will not execute immediately since there's another task in between which is generating test data.  Such task is long enough that makes room in running the client before it reaches Cypress tests step.

This make command is added as a convenience only for local development.  The task looks non-deterministic (maybe depending on the local machine) but I think it will work just fine.

Reply:
Alright. We will see how it goes.